docs/installation.md

```diff
@@ -10,7 +10,7 @@ Setting up gotify/server with docker is pretty easy, you basically just have to
 Before starting gotify/server you may read the [Configuration](configuration.md) if you f.ex. use a different database.
 
 ```bash
-$ docker run -p 80:80 -v /var/gotify/data:/app/data gotify/server
```

Comment:
I don't think this should be the default, some users may just want to try out gotify, and don't want it running in the background. Users that want to use the docker image can add the parameters themselves.